test/harnesses/EverlongTest.sol

```diff
@@ -111,6 +113,55 @@ contract EverlongTest is HyperdriveTest, IEverlongEvents {
     function depositEverlong(
         uint256 _amount,
         address _depositor
+    ) internal returns (uint256 shares) {
+        return
+            depositEverlong(
+                _amount,
+                _depositor,
+                true,
+                IEverlong.RebalanceOptions({
+                    spendingOverride: 0,
+                    minOutput: 0,
+                    minVaultSharePrice: 0,
+                    positionClosureLimit: 0,
+                    extraData: ""
+                })
+            );
+    }
+
+    function depositEverlong(
```

Comment:
You've commented several times that you wish Hyperdrive's testing tools and harnesses were better documented, and we've been adding Natspec to tools in the test suite per this request. Just calling this out since it doesn't seem like any of these functions are documented